types/redux-infinite-scroll/README.md

```diff
@@ -0,0 +1,16 @@
+# Installation
```

Comment:
We will generate this automatically for you. Don't include this on DefinitelyTyped.